packages/odyssey-react/src/components/TextInput/TextInput.module.scss

```diff
@@ -81,16 +80,25 @@
   }
 }
 
-.outer {
-  position: relative;
+.prefix {
+  color: var(--PrefixTextColor);
+  padding-inline-end: var(--PaddingInline);
+}
+
+.suffix {
+  color: var(--SuffixTextColor);
+  padding-inline-start: var(--PaddingInline);
 }
 
-.indicator {
-  position: absolute;
-  inset-block-start: 50%;
-  inset-inline-start: var(--IndicatorInsetInlineStart);
-  transform: translateY(-50%);
-  font-size: var(--IndicatorSize);
-  line-height: 1;
-  pointer-events: none;
+.input {
```

Comment:
This strips away the browser defaults from our `input` to make it as invisible as possible. Some browser-specific properties may be missing.